lib/plugin/github.js

```diff
@@ -18,7 +18,7 @@ function buildLinkUrl (repository, url) {
 // using the repository object provided
 function replaceTagAttribute (html, tag, attribute, buildUrl, repository) {
   // look for the attribute's value
-  var regex = new RegExp('^<\\s*' + tag + '[^>]*\\b' + attribute + '\\s*=\\s*', 'i')
+  var regex = new RegExp('<\\s*' + tag + '[^>]*\\b' + attribute + '\\s*=\\s*', 'i')
```

Comment:
This regex was originally anchored at `^` because in `html_inline` tokens, the inline tags always start at the beginning of the token content. But with `html_block` we might have a whole set of complex markup, so we can't do that for those.